playbooks/host/host-member-managedby_host-absent.yml

```diff
@@ -4,7 +4,7 @@
   become: true
 
   tasks:
-    ipahost:
+  - ipahost:
       ipaadmin_password: SomeADMINpassword
       name: host01.exmaple.com
       managedby_host: server.exmaple.com
```

Comment:
there is a typo here and in all other playbooks as well.

Reply:
This is an old issue, and appears in other playbooks as well. I'll leave it as is, and address in in other PR(s).